README.md

```diff
@@ -110,37 +110,40 @@ Otherwise :violin: kindly ask someone from the dev team to help troubleshoot.
+|-------------------------------------------------|----------------------------------------| -----------
+| DEFAULT_WEBSITE_URL                             | "http://localhost:8080"                | Service URL
+| DEFAULT_LEGACY_CACHE_API_URL                    | "http://localhost:29100"               | Cache API Service URL
+| DEFAULT_SLACK_WARNING_CHANNEL                   | "slack-client-test"                    | Slack channel
+| DEFAULT_SLACK_ALARM_CHANNEL                     | "slack-client-test"                    | Slack alarm channel
+| DEFAULT_SLACK_USERNAME                          | "Zebedee"                              | Slack user
+| DEFAULT_PUBLIC_WEBSITE_URL                      | "http://localhost:8080"                | Service public URL
+| DEFAULT_FLORENCE_URL                            | "http://localhost:8081"                | Florence URL
+| DEFAULT_BRIAN_URL                               | "http://localhost:8083"                | Brian URL
+| DEFAULT_TRAIN_URL                               | "http://localhost:8084"                | Train URL
+| DEFAULT_DYLAN_URL                               | "http://localhost:8085"                | Dylan URL
+| CONTENT_DIRECTORY                               | "content"                              | Content directory
+| KAFKA_SEC_PROTO                                 | _unset_                                | if set to "TLS", kafka connections will use TLS
+| KAFKA_SEC_CLIENT_CERT                           | _unset_                                | PEM for the client certificate [1]
+| KAFKA_SEC_CLIENT_KEY                            | _unset_                                | PEM for the client key [1]
+| MATHJAX_SERVICE_URL                             | "http://localhost:8888"                | Mathjax service URL
+| DATASET_API_URL                                 | "http://localhost:22000"               | Dataset API URL
+| IMAGE_API_URL                                   | "http://localhost:24700"               | Image API URL
+| ENABLE_KAFKA                                    | false                                  | Feature flag to send kafka messages when a collection is published
+| ENABLE_LEGACY_CACHE_API                         | false                                  | Feature flag to enable Cache API
+| KAFKA_ADDR                                      | "localhost:9092"                       | Comma seperated list of kafka brokers
+| KAFKA_CONTENT_UPDATED_TOPIC                     | content-updated                        | Kafka topic to send content updated event to
+| DATASET_API_AUTH_TOKEN                          | "FD0108EA-825D-411C-9B1D-41EF7727F465" | Dataset API authentication token
+| SERVICE_AUTH_TOKEN                              | "15C0E4EE-777F-4C61-8CDB-2898CEB34657" | Service API authentication token
+| LEGACY_CACHE_API_SERVICE_AUTH_TOKEN             | "...."                                 | Legacy Cache API authentication token
```

Comment:
Another question: should the name include the word "Service"? Shouldn't it be `LEGACY_CACHE_API_AUTH_TOKEN`. I'm saying that because there are a couple other tokens in here: the Dataset API one does not have the word "service" in it. The Service auth token may just refer to this service, i.e. Zebedee. Does that make sense? I could be wrong, so I'm happy to be corrected, it's not a big deal in any case.